frontend/individuals.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file This file contains the Individuals function. This function fetches and
+ *       displays a list of all individuals that the current user has access to.
+ */
+import React from 'react'
+import { forwardRef } from 'react';
+import MaterialTable from 'material-table'
+
+import AddBox from '@material-ui/icons/AddBox';
+import ArrowDownward from '@material-ui/icons/ArrowDownward';
+import Check from '@material-ui/icons/Check';
+import ChevronLeft from '@material-ui/icons/ChevronLeft';
+import ChevronRight from '@material-ui/icons/ChevronRight';
+import Clear from '@material-ui/icons/Clear';
+import DeleteOutline from '@material-ui/icons/DeleteOutline';
+import Edit from '@material-ui/icons/Edit';
+import FilterList from '@material-ui/icons/FilterList';
+import FirstPage from '@material-ui/icons/FirstPage';
+import LastPage from '@material-ui/icons/LastPage';
+import Remove from '@material-ui/icons/Remove';
+import SaveAlt from '@material-ui/icons/SaveAlt';
+import Search from '@material-ui/icons/Search';
+import ViewColumn from '@material-ui/icons/ViewColumn';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { useDataContext } from './data_context'
+import { Individual } from '~data_context_global';
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  table: {
+    height: "calc(100% - 75px)",
+    padding: "0 20px",
+    overflowY: "scroll",
+  },
+});
+
+const tableIcons = {
```

Comment:
Move all the icons stuff to an own file?

Reply:
Yeah - we can do that once the table is finished. In the mean time I'd prefer to keep them here so that I can look and change easier :)